neofetch

```diff
@@ -1644,6 +1644,43 @@ get_term_font() {
             term_font="$(osascript -e 'tell application "Terminal" to font name of window frontmost')"
         ;;
 
+        "iTerm2")
+            # Unfortunately the profile name is not unique, but it seems to be the only thing
+            # that identifies an active profile. There is the "id of current session of current window"
+            # thou, but that does not match to a guid in the plist.
+            # So, be warned! Collisions may occur!
+            # See: https://groups.google.com/forum/#!topic/iterm2-discuss/0tO3xZ4Zlwg
+            local currentProfileName
+            currentProfileName=$(osascript -e 'tell application "iTerm2" to profile name of current session of current window')
+
+            # Warning: Dynamic profiles are not taken into account here!
+            # https://www.iterm2.com/documentation-dynamic-profiles.html
+
+            # Count Guids in "New Bookmarks"; they should be unique
+            local profilesCount
+            profilesCount=$(/usr/libexec/PlistBuddy -c "Print :New\ Bookmarks:" ~/Library/Preferences/com.googlecode.iterm2.plist 2>/dev/null | grep -c "Guid")
+            for idx in $(seq 0 "${profilesCount}"); do
```

Comment:
We aren't using `seq` anywhere else in the script. Instead you can use a C style for loop which removes the call to an external process.

```sh
for ((i=0; i<=profiles_count; i++)); do
```